src/cli/remotes-wizard.js

```diff
@@ -8,21 +8,24 @@ export default async function (testCafe, remoteCount, showQRCode) {
 
     if (remoteCount) {
         log.hideSpinner();
+        if (remoteCount === 1)
+            log.write(`Connecting 1 remote browser...\nNavigate to the following URL from the remote browser.`);
+	else
+            log.write(`Connecting ${remoteCount} remote browsers...\nNavigate to the appropriate URL from each of the remote browsers.`);
+        if (showQRCode)
+            log.write('You can either enter the URL or scan the QR-code.');
 
         for (var i = 0; i < remoteCount; i++) {
             var browserConnection = testCafe.createBrowserConnection();
 
-            log.write(`To connect a remote browser #${i + 1}, use it to open ${chalk.underline.blue(browserConnection.url)}`);
-
-            if (showQRCode) {
-                log.write('or scan this QR-code:\n');
+            log.write(`\nBrowser #${i + 1}: ${chalk.underline.blue(browserConnection.url)}`);
+            if (showQRCode)
```

Comment:
Need blank line here. Don't insert new line here: ``\nBrowser #${i + 1}:`. Just keep it as it was